test/readwrite.jl

```diff
@@ -1,7 +1,9 @@
-using TimeSeries, MarketData
+using FactCheck
```

Comment:
In the future, please don't make changes in one test file that isn't reflected in a different test files and submit that as PR. It introduces inconsistencies. In this case, I'll merge the PR and remedy it.